packages/assets-controllers/src/NftController.ts

```diff
@@ -512,6 +512,53 @@ export class NftController extends BaseController<
     });
   }
 
+  /**
+   * Fetches NFT Collection Metadata from the NFT API.
+   *
+   * @param contractAddresses - The contract addresses of the NFTs.
+   * @param chainId - The chain ID of the network where the NFT is located.
+   * @returns NFT collections metadata.
+   */
+  async fetchNftCollectionsMetadata(
```

Comment:
I think this should short circuit if a chain id is provided that the NFT API doesn't support, to save a request + time.

Reply:
Good call @bergeron 
I couldn't see any endpoint / constants that I can use it. Can you elaborate how do I derive that information here?